sources/seafile_api_loader.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Source YunoHost helpers
+source /usr/share/yunohost/helpers
+
+# Stop script if errors
+#ynh_abort_if_errors
```

Comment:
Well, why do you disable this ?

Reply:
I'm not sure this hook handle well actions, but we can try with it.